test/tests/ver-1e/comparison_ver-1e.py

```diff
@@ -4,28 +4,50 @@
 from matplotlib import gridspec
 import pandas as pd
 from scipy import special
+from numpy import sin,tan,sqrt,exp
 
 # ========= Comparison of concentration as a function of time ===================
 
 fig = plt.figure(figsize=[6.5, 5.5])
 gs = gridspec.GridSpec(1, 1)
 ax = fig.add_subplot(gs[0])
 
-tmap_sol = pd.read_csv("./ver-1e_csv.csv")
+tmap_sol = pd.read_csv("./gold/ver-1e_csv.csv")
 tmap_time = tmap_sol['time']
 tmap_conc = tmap_sol['conc_point1']
 
-analytical_sol = pd.read_csv("./analytical_time.csv")
-analytical_time = analytical_sol['t']
-analytical_conc = analytical_sol['u']
+#Analytical solution
+t = np.expand_dims(tmap_time,axis=0)
+x = 15.75e-6
+c0 = 50.7079            # concentration at the PyC free surface (moles/um^3)
+a  = 33e-6              # thickness of the PyC layer (um)
+l  = 66e-6              # thickness of the SiC layer (um)
+D_PyC = 1.274e-7        # diffusivity in PyC (m^2/s)
+D_SiC = 2.622e-11       # diffusivity in SiC (m^2/s)
+
+k = sqrt(D_PyC/D_SiC)
+
+#Calculate lambda values for analytical solution
+lambda_range = np.arange(1e-12,1e5,1e-4)
+f = 1/tan(lambda_range*a)
+g = 1/tan(k*l*lambda_range)/k
```

Comment:
Yep, that's what we talked about: the equation in ver-1e in TMAP7 was incorrect, whereas TMAP4 was correct. Could you make a note in the documentation (like I did for ver-1a) to explain the typo in the TMAP7 V&V report to users?